code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -890,3 +898,52 @@ obj/item/asteroid/basilisk_hide/New()
 /mob/living/simple_animal/hostile/asteroid/pillow/death()
 	overlays.Cut()
 	..()
+
+/mob/living/simple_animal/hostile/retaliate/goat/wooly
+	name = "wooly goat"
+	desc = "An absolutely fierce-tempered beast that prefers cold climates. It has a very tough hide."
+	icon_state = "woolygoat"
+	icon_living = "woolygoat"
+	icon_dead = "woolygoat_dead"
+	gives_milk = FALSE
+	health = 120
+	maxHealth = 120
+	anger_chance = 4 //4% per tick to get angry
+	faction = "mining"
+
+/mob/living/simple_animal/hostile/retaliate/goat/wooly/New()
+	..()
+	gender = pick(MALE,FEMALE)
+	gives_milk = (FEMALE ? TRUE : FALSE)
+
+/mob/living/simple_animal/hostile/retaliate/goat/wooly/Life()
```

Comment:
a mistake seems to have been made here, Life got pasted by mistake, and anger_chance should be used in the parent Life()